test/units/TEST-75-RESOLVED.sh

```diff
@@ -66,11 +68,14 @@ restart_resolved() {
 }
 
 # Test for resolvectl, resolvconf
+manual_testcase_01_resolvectl() {
```

Comment:
I know that the `manual_` prefix for test cases that are called before the setup phase (instead of being run as a part of `run_testcases`) is not great, but this was the best I came up with. If you have any better suggestion, feel free to let me know. :)